private-endpoint.tf

```diff
@@ -6,7 +6,7 @@ locals {
 
 data "azurerm_subnet" "private-endpoints" {
   count    = var.enable_private_endpoint && local.use_default_subnet_id ? 1 : 0
-  provider = azurerm.private-endpoint-subnet
+  provider = azurerm.private-endpoint
```

Comment:
any chance we can use `private_endpoint` or too late?

Reply:
ah yeah, no worries I can update the pr's